site/src/templates/standard-article.js

```diff
@@ -144,6 +158,8 @@ export default class Article extends React.Component {
       })
       .filter(peer => peer !== undefined)
 
+      let paginatedEdges = this.state.paginationObject.filterResults(peerEdges)
```

Comment:
Would prefer paginatedPeerEdges to clarify this is just a pagination facade over the peerEdges collection.

Reply:
Done